src/pages/guidelines/2x-grid/implementation.mdx

```diff
@@ -0,0 +1,1264 @@
+
+![Do keep all type (inside and outside containers) aligned with the column grid.](images/2px_do.png)
+
+  </DoDontExample>
+</Column>
+<Column colMd={4} colLg={4} >
+  <DoDontExample caption="Do not hang type into the gutter under any circumstances.">
+
+![Do not hang type into the gutter under any circumstances.](images/2px_dont.png)
+
+  </DoDontExample>
+</Column>
+</Row>
+
+### Mix and match
+
+You can use rows to mix and match grid modes. In this situation, margin bottom between the rows can vary. Because of input fields with labels, both the Cozy grid mode and the Condensed grid are most often used in conjunction with the Comfortable grid mode.
+
+#### Usage
+
+Unless you're using the Comfortable grid mode exclusively, mixing and matching is going to be the rule rather than the exception on most platforms — especially since our labeled components (input fields, dropdowns etc.) require a 32px gutter. We've provided a couple of examples below and will keep adding helpful uses cases as we find them.
+
+![Mixed use](images/Mixed_use_revised.png)
+
+<Caption>
+  1. Condensed grid mode <br /> 2. Comfortable grid mode{' '}
+</Caption>
+
+<br /> 
+<br />
+
+<Row>
+<Column colMd={4} colLg={4}>
+  <DoDontExample type="do" caption="Do apply different grid modes to different components (like input fields with labels and tiles).">
+
+![Do apply different grid modes to different components (like input fields with labels and tiles).](images/Mixed_use_do.png)
+
+  </DoDontExample>
+</Column>
+<Column colMd={4} colLg={4} >
+  <DoDontExample caption="Do not apply different grid modes to the same component on the same page.">
+
+![Do not apply different grid modes to the same component on the same page.](images/Mixed_use_dont.png)
+
+  </DoDontExample>
+</Column>
+</Row>
+
+![Mixed use 2](images/Mixed_use_2.png)
+
+<Caption>
+  1. Condensed grid mode <br /> 2. Comfortable grid mode{' '}
+</Caption>
+
+<br /> 
+<br />
+
+<Row>
+<Column colMd={4} colLg={4}>
+  <DoDontExample type="do" caption="Do apply different grid modes to different components (like input fields with labels and tiles).">
+
+![Do apply different grid modes to different components (like input fields with labels and tiles).](images/Mixed_use_do_2.png)
+
+  </DoDontExample>
+</Column>
+<Column colMd={4} colLg={4} >
+  <DoDontExample caption="Do not apply different grid modes to the same component on the same page.">
+
+![Do not apply different grid modes to the same component on the same page.](images/MIxed_use_dont_new.png)
+
+  </DoDontExample>
+</Column>
+</Row>
+
+## Incorporating the Shell
+
+We cover more about shell behavior on the Layout Overview tab in the [Grid influencers](/guidelines/layout/overview#grid-influencers) section. However, we've had a lot of questions about how the shell works in conjunction with our grid. That's why we recently incorporated the shell into our grid tooling as a Sketch template. We'd also like to show it in action here in a product.
```

Comment:
- Need to fix link to "grid influencers" 
- Can we hyperlink to the Sketch template: https://sketch.cloud/s/EjVmA